brainstormforce/ultimate-addons-for-gutenberg
# Fix: Content Timeline - connector layout on mobile devices is not displaying properly.

[Trello](https://trello.com/c/4nVk8zWr/279-fix-content-timeline-connector-layout-on-mobile-devices-is-not-displaying-properly)

### Description
Content timeline connector not looking properly on mobile devices.

### Test Cases
1. Drag and drop content timeline.
2. Check on mobile devices on the frontend is it displaying properly or not

### Screenshots
Issue ->  - https://share.bsf.io/llu55qoj

### Types of changes
- Updated CSS for connector

### How has this been tested?
Checked whether it is displaying properly or not.

### Checklist:
